defaultentities.js

```diff
@@ -46,6 +46,154 @@ module.exports = { // ADD MORE (vanilla) AS YOU PLEASE (or modded if it's just f
     width: 1,
     height: 2
   },
+  
+  basic_underground_belt: {
+    type: item,
+    width: 1,
+    height: 1
+  },
+  fast_underground_belt: {
+    type: item,
+    width: 1,
+    height: 1
+  },
+  express_underground_belt: {
+    type: item,
+    width: 1,
+    height: 1
+  },
+  basic_splitter: { // how do we deal with rotatable items?
```

Comment:
They are already handled, you just need the default width and height when you bring it out.